backend/cfg/cfg.ml

```diff
@@ -227,40 +236,26 @@ let dump_op ?(specific = fun ppf _ -> Format.fprintf ppf "specific") ppf =
   | Compf _ -> Format.fprintf ppf "compf"
   | Floatofint -> Format.fprintf ppf "floattoint"
   | Intoffloat -> Format.fprintf ppf "intoffloat"
-  | Specific op -> specific ppf op
-  | Probe { name; handler_code_sym } ->
-    Format.fprintf ppf "probe %s %s" name handler_code_sym
+  | Specific _ -> Format.fprintf ppf "specific"
   | Probe_is_enabled { name } -> Format.fprintf ppf "probe_is_enabled %s" name
   | Opaque -> Format.fprintf ppf "opaque"
   | Begin_region -> Format.fprintf ppf "beginregion"
   | End_region -> Format.fprintf ppf "endregion"
   | Name_for_debugger _ -> Format.fprintf ppf "name_for_debugger"
 
-let dump_call ppf = function
-  | P prim_call -> (
-    match prim_call with
-    | External { func_symbol : string; _ } ->
-      Format.fprintf ppf "external %s" func_symbol
-    | Alloc { bytes : int; _ } -> Format.fprintf ppf "alloc %d" bytes
-    | Checkbound _ -> Format.fprintf ppf "checkbound")
-  | F func_call -> (
-    match func_call with
-    | Indirect -> Format.fprintf ppf "indirect"
-    | Direct { func_symbol : string; _ } ->
-      Format.fprintf ppf "direct %s" func_symbol)
-
 let dump_basic ppf (basic : basic) =
   let open Format in
   match basic with
   | Op op -> dump_op ppf op
-  | Call call -> fprintf ppf "Call %a" dump_call call
   | Reloadretaddr -> fprintf ppf "Reloadretaddr"
   | Pushtrap { lbl_handler } -> fprintf ppf "Pushtrap handler=%d" lbl_handler
   | Poptrap -> fprintf ppf "Poptrap"
   | Prologue -> fprintf ppf "Prologue"
 
-let dump_terminator' ?(print_reg = Printmach.reg) ?(args = [||]) ?(sep = "\n")
-    ppf (terminator : terminator) =
+let dump_terminator' ?(print_reg = Printmach.reg) ?(print_res = fun _ -> ())
```

Comment:
I've made such that we pass the results array.
I don't know why I did it by passing `print_res`.

About the printing change, it was required to be here because before there weren't any terminators that had results so this is rather a fix corresponding to the new layout rather than standalone improvement.